src/data/payments/data.ts

```diff
@@ -828,6 +829,50 @@ export interface PaysafecardDetails {
   customerReference: string;
 }
 
+export interface PointOfSaleDetails {
```

Comment:
In the [method specific details for POS](https://docs.mollie.com/reference/extra-payment-parameters#point-of-sale-1) I can only see `terminalId`.

Are the other properties also included? Are they all non-optional?

Reply:
To clarify: This relates to line 839 to 875.
These were part of the original PR for this update, but I cannot confirm where these properties came from or if they are accurate.

Reply:
- Details
	- terminalId
	- cardNumber
	- cardAudience
	- cardLabel
		- Possible values limited to Visa or Mastercard for terminal transactions. Maestro is currently labeled as Mastercard
	- cardCountryCode
	- cardFingerprint